sandbox/apparmor/apparmor.go

```diff
@@ -51,6 +53,30 @@ const (
 	Full
 )
 
+func setupConfCacheDirs(newrootdir string) {
```

Comment:
This part looks okay.

Note that solus does something weird that we should look at one day. I think they have a even-different setup.

Reply:
if the spread test fails on this pr or the one after this, I will push up a TODO for looking at solus here